### DISCUSSION TITLE
Profiles according choosen aircraft

### DISCUSSION THREAD

**Discussion by JoergM-D** [nguyenquyhy/Flight-Tracker-StreamDeck#276] [Ideas]
Hi,
is it possible?
Creating separate profiles for each aircraft-type and they start  automatically, by choosing the aircraft inside the sim? The MSFS trigger the right profile on Streamdeck.

Cheers Jörg

**Comment by nguyenquyhy** [comment]
In a user friendly way, nope. Stream Deck allows plugin to switch to only prebuilt profiles included in the installation packages. Those profiles are created at plugin installation and read-only.

Some hacks have been suggested such as the plugin pre-create a bunch of empty profiles, then you can change Stream Deck database to swap in your own profile to get around the read-only restriction. Or you can change the plugin manifest files to trick Stream Deck into thinking some existing profiles are actually included in the plugin installation. Unfortunately, all those are very troublesome for user and very unstable so I will not include in the plugin.

**Reply by JoergM-D** [reply]
Thanks for your quick response.
Then I'll create profiles for each plane and choose them manualy.

Cheers 
Jörg
